plugins/qt3dinspector/geometryextension/buffermodel.cpp

```diff
@@ -83,16 +83,18 @@ void BufferModel::updateAttributes()
 
 void BufferModel::updateAttribute(const GammaRay::Qt3DGeometryAttributeData &attrData)
 {
-    m_rowSize = m_buffer.size() / attrData.count;
```

Comment:
@vkrause: The update of `m_rowSize` in this method looks odd. Is that correct?

In case `updateAttribute` is called multiple times in `updateAttributes` (is that possible?), the value is overwritten. What am I missing?